lib/diagnostic.js

```diff
@@ -20,7 +31,18 @@ const sum = function sum() {
 // Write a function that returns an array of the specified length
 // with all the elements set to defaultValue
 const newArray = function newArray(length, defaultValue) {
-
+  let newArr = [];
+  if(defaultValue instanceof Array) {
+    for(let i = 0; i < length; i++) {
+      newArr[i] = defaultValue[i];
+    }
+    return newArr;
```

Comment:
This doesn't really address what `newArray` is supposed to do - create new array of size `length`, populated with some default value.